HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Dashboard_page.Steps.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using HealthMonitoring.AcceptanceTests.Helpers;
+using HealthMonitoring.AcceptanceTests.Helpers.Selenium;
+using LightBDD;
+using OpenQA.Selenium;
+using RestSharp;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace HealthMonitoring.AcceptanceTests.Scenarios.Selenium
+{
+    public partial class Dashboard_page : FeatureFixture, IDisposable
+    {
+        private readonly RestClient _client;
+        private readonly IWebDriver _driver;
+        private readonly string _healthy = EndpointStatus.Healthy.ToString().ToLower();
+        private readonly string _faulty = EndpointStatus.Faulty.ToString().ToLower();
+        private readonly string _dashboardUrl = $"{SeleniumConfiguration.BaseUrl}dashboard?endpoint-frequency=1000000&config-frequency=1000000";
+
+        public Dashboard_page(ITestOutputHelper output) : base(output)
+        {
+            _driver = SeleniumConfiguration.GetWebDriver();
+            _client = ClientHelper.Build();
+            _client.RegisterTestEndpoints();
+        }
+
+        public void Load_dashboard_page()
+        {
+            _driver.LoadUrl(_dashboardUrl);
+        }
+
+        public void Click_on_menu_header_should_redirect_to_home_page()
```

Comment:
please follow the structure of other tests and write it in lightbdd format, with explicit given-when-then steps